api/libproject/devstart.rb

```diff
@@ -2490,12 +2508,12 @@ def create_project_resources(gcc)
 end
 
 def setup_project_data(gcc, cdr_db_name)
-  root_password, workbench_password = random_password(), random_password()
+  root_password, workbench_password, readonly_password = random_password(), random_password(), random_password()
```

Comment:
Please break this into three assignments, one on each line, or make a method that outputs an array of random passwords so it's like `root_password, workbench_password, readonly_password = *random_passwords(3)`. I stared at it for a good minute before I realized it was a multi-assignment.

Reply:
sure